web/src/js/components/Widget/Widgets/Bookmarks/BookmarkChip.js

```diff
@@ -128,51 +120,60 @@ class BookmarkChip extends React.Component {
     }
     return (
       <span>
-        <Paper
-          zDepth={0}
-          style={{
-            position: 'relative',
-            display: 'flex',
-            justifyContent: 'center',
-            cursor: 'pointer',
-            alignItems: 'center',
-            margin: 5,
-            minWidth: 84,
-            maxWidth: 150,
-            height: 50,
-            fontSize: 14,
-            padding: 10,
-            backgroundColor: bookmarkColor,
-            color: '#FFF',
-            userSelect: 'none',
-          }}
-          onClick={this.onClick.bind(this)}
+        <ConditionalWrapper
+          condition={!this.props.editMode}
+          wrapper={children => (
+            <Link to={this.addProtocolToURLIfNeeded(this.props.bookmark.link)}>
```

Comment:
Please add a `target="_top"` prop to Link, because the new tab page frequently renders in an iframe.